src/with-urql-client.tsx

```diff
@@ -47,15 +46,11 @@ function withUrqlClient<T = any, IP = any>(
     fetchExchange,
   ],
 ) {
-  return (
-    Page:
-      | NextComponentClass<T & IP & WithUrqlClient, IP>
-      | NextFC<T & IP & WithUrqlClient, IP>,
-  ) => {
-    const withUrql: NextFC<
-      T & IP & WithUrqlClient & WithUrqlInitialProps & PageProps,
+  return (Page: NextPage<T & IP & WithUrqlClient, IP>) => {
+    const withUrql: NextComponentType<
```

Comment:
Update `NextFC` to `NextComponentType`, who's type variables are in the reverse order of `NextFC`